src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -1,22 +1,64 @@
 package frc.robot.subsystems;
 
-import com.ctre.phoenix.motorcontrol.FollowerType;
+import com.ctre.phoenix.motorcontrol.ControlMode;
 import com.ctre.phoenix.motorcontrol.InvertType;
 import com.ctre.phoenix.motorcontrol.VictorSPXControlMode;
-import com.ctre.phoenix.motorcontrol.can.VictorSPX;
+import com.ctre.phoenix.motorcontrol.can.WPI_VictorSPX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.DigitalInput;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj2.command.button.Trigger;
+
 import static frc.robot.Constants.*;
 
-public class IntakeSubsystem {
-    private VictorSPX upperIntakeMotor = new VictorSPX(UPPER_INTAKE_MOTOR);
-    private VictorSPX bottomIntakeMotor = new VictorSPX(BOTTOM_INTAKE_MOTOR);
+public class IntakeSubsystem extends SubsystemBase {
+    private WPI_VictorSPX upperIntakeMotor = new WPI_VictorSPX(UPPER_INTAKE_MOTOR);
+    private CANSparkMax bottomIntakeMotor = new CANSparkMax(BOTTOM_INTAKE_MOTOR, MotorType.kBrushed);
```

Comment:
I think it's because sometimes we just want to run either the upper or lower but not both. Since the upper tends to jam if we over spin it, we'd just use the lower in some cases.

Reply:
I meant different motor controllers. One is a spark and one is a victor

Reply:
Oh yes, we do have separate ones lol. I think that was the last SparkMax too.